src/mame/drivers/segas32.cpp

```diff
@@ -1081,13 +1081,15 @@ WRITE8_MEMBER(segas32_state::sound_bank_hi_w)
 
 WRITE8_MEMBER(segas32_state::multipcm_bank_w)
 {
-	m_multipcm->set_bank(0x80000 * ((data >> 3) & 7), 0x80000 * (data & 7));
+	membank("multipcmbankhi")->set_base(memregion("sega")->base() + (0x80000 * ((data >> 3) & 7)));
```

Comment:
There are too many tag lookups here. Please revise this and the other drivers using banked MultiPCM to use `required_memory_bank` finders and configured banking.

Reply:
also configured banking tends to be preferred these days when possible, rather than the ->set_base type

search the source for uses of
	->configure_entries(
and
	->set_entry(0);
if you're unfamiliar with it

don't know if it's appropriate here tho, some cases it doesn't really fit as well.